tqdm/_utils.py

```diff
@@ -1,5 +1,12 @@
 import os
 import subprocess
+from platform import system as _curos
+CUR_OS = _curos()
+IS_WIN = CUR_OS in ['Windows', 'cli']
+IS_NIX = (not IS_WIN) and any(
+    CUR_OS.startswith(i) for i in
+    ['CYGWIN', 'MSYS', 'Linux', 'Darwin', 'SunOS', 'FreeBSD'])
```

Comment:
if I were you I would have added `.lower()` and kept actual literals for comparison in lower case.

-- said yarikoptic and then looked at his own code which doesn't (yet) do that ;)